Add the solver interThermalRadConvFoam

BiElisa opened the pull request.

gerlero commented:
@BiElisa thanks for submitting your package to the OPI!

Besides the requested change in the `"version"` field, can you rename the `interThermalRadConvFoam` directory to all lowercase (i.e. `interthermalradconvfoam`)? OPI packages must be all-lowercase.

Note that the `styro` package manager itself is case insensitive, so `interThermalRadConvFoam` will then work too for actually installing the package.

BiElisa commented:
Hi @gerlero ! 
Thanks for your message! I've changed the name of the directory and corrected  the "version" field.
I hope this time everything is fine!
All the best,
Elisa

gerlero commented:
@BiElisa Thanks! The validation workflow is failing due to the description being longer than 200 characters. Would you mind shortening the description to fit this limit?

BiElisa commented:
@gerlero I've shortened the description, let's see if it works now! :)

gerlero commented:
@BiElisa Your package should now be installable with [`styro`](https://github.com/gerlero/styro) via:

```bash
styro install interthermalradconvfoam
```

Thanks again for contributing!

BiElisa commented:
Wow! Thanks to you!!

I have a further question, maybe you know how to help me! I'm trying to create a badge with shields, but I find a problem. 
I add my URL  https://github.com/exasim-project/opi/blob/main/pkg/interthermalradconvfoam/metadata.json and the query you suggest in the readme $.[name,type,version], but I obtain a badge that contains 14 names from the list of all packages.

Thanks for any help!
Best,
Elisa

![badge](https://github.com/user-attachments/assets/3da94947-fdce-4549-b2f1-298415b72ece)

greole commented:
@BiElisa, weird. I guess i'll create an issue for that.
